nullaway/src/test/java/com/uber/nullaway/NullAwayAutoSuggestTest.java

```diff
@@ -37,7 +38,7 @@ public class NullAwayAutoSuggestTest {
 
   @Rule public TemporaryFolder temporaryFolder = new TemporaryFolder();
 
-  private ErrorProneFlags flags;
+  public ErrorProneFlags flags;
```

Comment:
Why is this change to public needed?

Reply:
Good catch! It isn't, it was from a different try at changing `makeTestHelper`. Let me change it back.